ibm_watson/speech_to_text_v1_adapter.py

```diff
@@ -251,11 +254,7 @@ def recognize_using_websocket(self,
         options = {k: v for k, v in options.items() if v is not None}
         request['options'] = options
 
-        RecognizeListener(audio,
-                          request.get('options'),
-                          recognize_callback,
-                          request.get('url'),
-                          request.get('headers'),
-                          http_proxy_host,
-                          http_proxy_port,
+        RecognizeListener(audio, request.get('options'), recognize_callback,
+                          request.get('url'), request.get('headers'),
+                          http_proxy_host, http_proxy_port,
                           self.disable_ssl_verification)
```

Comment:
why reformat this? I think it was more readable before

Reply:
Used the same `yapf` formatter that is used for generated files.